Broadcasting.sh

```
@@ -0,0 +1,436 @@
+#!/usr/bin/env python3
```

Shellcheck (0.10.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)